packages/frontend-api/src/Model/Resolver/Order/OrderResolver.php

```diff
@@ -32,19 +33,27 @@ class OrderResolver implements ResolverInterface, AliasedInterface
      */
     protected $domain;
 
+    /**
+     * @var \Shopsys\FrontendApiBundle\Model\Order\OrderFacade
+     */
+    protected $frontendApiOrderFacade;
+
     /**
      * @param \Shopsys\FrameworkBundle\Model\Customer\User\CurrentCustomerUser $currentCustomerUser
      * @param \Shopsys\FrameworkBundle\Model\Order\OrderFacade $orderFacade
      * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
+     * @param \Shopsys\FrontendApiBundle\Model\Order\OrderFacade $frontendApiOrderFacade
      */
     public function __construct(
         CurrentCustomerUser $currentCustomerUser,
         OrderFacade $orderFacade,
-        Domain $domain
+        Domain $domain,
+        FrontendApiOrderFacade $frontendApiOrderFacade
     ) {
         $this->orderFacade = $orderFacade;
```

Comment:
I am not sure if original `OrderFacade` should be used here.

It is a bit confusing having 2 different order facades. This one is only used on line 63 `$this->orderFacade->getByUrlHashAndDomain`. Maybe this method could be exposed into FEAPI Order Facade too?

Reply:
I think this is absolutely ok. Those are two different facades and we do not want to duplicate code in FE API, we just wanna prevent methods from FE API to appear in framework.

Reply:
I do not want to duplicate the logic I was more thinking about bridging this method through FEAPI Order Facade...

It will be hard to think about what facade should I use. It is sad we cannot use the advantages of inheritance here...

Reply:
I don't think it's a problem. 
We cannot use inheritance, because then it would be hard to extend the proper facade. 

I would like to not composite facades (like `$frontendApiOrderFacade->getOrderFacade()->...`).

Reply:
As `FrontendApiOrderFacade` is already dependent on `OrderFacade` it is useless and confusing as a delevoler to have both dependencies in `OrderResolver` - when one would do just fine (easier developmnet, testing, more coupled).

I was not suggesting making composition from the resolver. I was more thinking about exposing the method in `FrontendApiOrderFacade` like:

```php
// OrderResolver

$this-frontendApiOrderFacade->getMeAny();
```

```php
// FrontendApiOrderFacade

public function getMeAny(): Any
{
    return $this->orderFacade->getMeAny();
}
```